Dockerfile

```diff
@@ -5,8 +5,7 @@ WORKDIR /app
 RUN apk add --no-cache \
     postgresql-client \
     nodejs \
-    git \
-    graphicsmagick
+    git
```

Comment:
Why have you removed graphicsmagick from here?

Reply:
I didnt think it was required if its being installed in the test, although come to think of it, is it required for the proper build for the binary?

Reply:
It is